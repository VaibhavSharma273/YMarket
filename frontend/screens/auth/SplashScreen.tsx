import { Image, StyleSheet } from 'react-native';
import { View } from '../../components/Themed';

export default function SplashScreen() {
    return (
      <View style={styles.container}>
        <Image source={require('../../assets/images/ymarket_logo.png')} style={styles.image} />
      </View>
    );
  }

const styles = StyleSheet.create({
    container: {
      flex: 1,
      alignItems: 'center',
      justifyContent: 'center',
    },
    image: {
        width: 85,
        height: 75,
        marginBottom: 10,
      },
  });
  