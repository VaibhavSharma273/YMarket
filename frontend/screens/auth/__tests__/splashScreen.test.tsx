import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import SplashScreen from '../SplashScreen';
import { createTestProps } from '../../../jest.setup';
import renderer from 'react-test-renderer';

describe('Confirmation Screen component', () => {
    let component: ShallowWrapper;
    let props: any;
    beforeEach(() => {
        props = createTestProps({});
        component = shallow(<SplashScreen {...props} />);
    });
    it(`renders correctly`, () => {
        expect(component.length).toBe(1);
    });
    it('matches the snapshot', () => {
        const tree = renderer.create(<SplashScreen {...props} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
