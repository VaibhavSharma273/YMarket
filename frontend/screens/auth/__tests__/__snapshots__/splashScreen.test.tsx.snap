// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmation Screen component matches the snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 75,
        "marginBottom": 10,
        "width": 85,
      }
    }
  />
</View>
`;
